week2/helpMenu.txt

```diff
@@ -0,0 +1,35 @@
+
+    HELP MENU                                                                                    ||
```

Comment:
I like the way your help menu is nice and thorough!

There's a bit of a pattern for how to design CLI help menus - I wouldn't have expected you to know this, but you can check some examples for inspiration.

Try `git help`, `curl --help`, `cd --help`, `vim --help`, etc.